docs/docs/standard-lib/math.md

```diff
@@ -27,19 +27,19 @@ import Math;
 
 ### Constants
 
-| Constant     | Description                                            |
-|--------------|--------------------------------------------------------|
-| Math.pi      | The mathematical constant: 3.14159265358979            |
-| Math.e       | The mathematical constant: 2.71828182845905            |
-| Math.phi     | The mathematical constant: 1.61803398874989            |
-| Math.sqrt2   | The mathematical constant: 1.41421356237309            |
-| Math.sqrte   | The mathematical constant: 1.61803398874989            |
-| Math.sqrtpi  | The mathematical constant: 1.77245385090551            |
-| Math.sqrtphi | The mathematical constant: 1.27201964951406            |
-| Math.ln2     | The mathematical constant: 0.69314718055994            |
-| Math.ln10    | The mathematical constant: 2.30258509299404            |
-
-### Math.min(iterable)
+| Constant     | Description                                 |
+| ------------ | ------------------------------------------- |
+| Math.pi      | The mathematical constant: 3.14159265358979 |
+| Math.e       | The mathematical constant: 2.71828182845905 |
+| Math.phi     | The mathematical constant: 1.61803398874989 |
+| Math.sqrt2   | The mathematical constant: 1.41421356237309 |
+| Math.sqrte   | The mathematical constant: 1.61803398874989 |
+| Math.sqrtpi  | The mathematical constant: 1.77245385090551 |
+| Math.sqrtphi | The mathematical constant: 1.27201964951406 |
+| Math.ln2     | The mathematical constant: 0.69314718055994 |
+| Math.ln10    | The mathematical constant: 2.30258509299404 |
+
+### Math.min(Iterable) -> Number
```

Comment:
```suggestion
### Math.min(...Number|List) -> Number
```

What do you think about this notation over Iterable? Iterable isn't really true because in theory things like strings would be iterables

Reply:
I kinda like the way we're doing result.

```sh
Math.min(List<Number>) -> Number

Reply:
I was using that as like an OR as Math.min(1, 2, 3); is a fine operation, along with the list variant Math.min([1, 2, 3]); if you get what I mean